### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by Dairoe** [wojtryb/Shortcut-Composer#55] [Help me]
for the temp eraser while holding and changing eraser brush size it should not change the the drawing brush

**Comment by wojtryb** [comment]
You can synchronize/de-synchronize brush and eraser size, you can go to brush settings (F5) and change the "Eraser switch size" checkbox in bottom left corner.

If that is not what you want, you need to explain your problem more clearly.

**Comment by Dairoe** [comment]
after using krita for so long never knew that was there i see.
anyways thanks for the temp hold eraser feat idk why krita doesn't just add it by default
